src/job/jobHandler.ts

```diff
@@ -614,6 +615,24 @@ export abstract class JobHandler {
   public getLogger(): ILogger {
     return this._logger;
   }
+
+  protected async previewWebhook(): Promise<AxiosResponse> {
```

Comment:
NIT: I don't think the `AxiosResponse` is the correct type. I believe this corresponds to the entire response we get from `axios.post`, and not the `data` property. This is the interface definition for `AxiosResponse`:

```ts
export interface AxiosResponse<T = any>  {
  data: T;
  status: number;
  statusText: string;
  headers: any;
  config: AxiosRequestConfig;
  request?: any;
}
```

If we know what the response data should be, we can define our own interface, and provide it as a type argument in our `axios.post` call. Here's an example of what that could look like:

```ts
interface GatsbyCloudBuildResponse {
  success: boolean;
  staging_url: string;
}


protected async previewWebhook(): Promise<GatsbyCloudBuildResponse> {
  const previewWebhookURL = 'https://webhook.gatsbyjs.com/hooks/data_source';
  const gatsbySiteDataSource = process.env.GATSBY_CLOUD_PREVIEW_WEBHOOK;
  const url = `${previewWebhookURL}/${gatsbySiteDataSource}`;

  // the type for data will now be GatsbyCloudBuildResponse
  const { data } = await axios.post<GatsbyCloudBuildResponse>(
    url,
    {
      project: this.currJob.payload.project,
      branch: this.currJob.payload.branchName,
    },
    {
      headers: { 'x-gatsby-cloud-data-source': 'gatsby-source-snooty' },
    }
  );

    return data;
  }
```